mstm/__init__.py

```diff
@@ -210,6 +214,46 @@ def calc_intensity(target, incident, theta, phi):
                                          scat_mat_data[:,:,4]*incident.stokes_vec[2] +
                                          scat_mat_data[:,:,5]*incident.stokes_vec[3])                                  
     return intensity_data
+    
+def calc_cross_section(target, incident, theta, phi):
+    """
+    Calculate the cross section from wavelength. 
+    If theta = 0-180 and phi = 0-360, the cross section calculated is the total cross section
+    If theta = 90-180 and phi = 0-360, the cross section caclulated is the backscattering cross section,
+    where the backscattering cross section is proportional to the reflectance
+
+    Parameters
+    ----------
+    target: an object of the Target class
+    incident: an object of the Incident class
+    theta: numpy array of values for the polar scattering angle theta for scattering
+        matrix computations, in degrees (must be 0-180)
+    phi: numpy array of values for the azimuth angle theta for scattering matrix
+        computations, in degrees (must be 0-360)
+
+    Returns
+    -------
+    numpy array:
+        mstm_cscat_1
+    """
+    intensity_dat_1 = calc_intensity(target, incident, theta, phi)       
```

Comment:
I think we can do this much more efficiently.  Try first calculating the values on the phi, theta grid.  Then use scipy.interpolate.interp2d to generate an interpolating function.  Then use dblquad to integrate the interpolating function.  That should hopefully speed things up (note that because you're using the interpolating function and not calling the mstm.exe executable, dblquad() should work quickly).